bedrock/firefox/views.py

```diff
@@ -372,7 +372,14 @@ def get_template_names(self):
         version = self.kwargs.get('version') or ''
         locale = l10n_utils.get_locale(self.request)
 
-        if show_devbrowser_firstrun(version):
+        # variant is present for growth tests
+        variant = self.request.GET.get('f', None)
```

Comment:
Whilst this works, I believe we have a dedicated helper for such params, e.g. `funnelcake_param(self.request)`?

Worth adding a couple of tests for this as well for good measure?